src/Components/SignUpPage.jsx

```diff
@@ -0,0 +1,147 @@
+import { useState } from "react";
+import "./signUpPage.scss";
+import { account, ID } from "../lib/appwrite";
+import { validateEmailWithRegex } from "../lib/utils/validateEmailWithRegex";
+import { validatePasswordWithRegex } from "../lib/utils/validatePasswordWithRegex";
+
+export default function SignUpPage() {
+  const [showPassword, setShowPassword] = useState(false);
+  const [nameError, setNameError] = useState(true);
+  const [emailError, setEmailError] = useState(true);
+  const [passwordError, setPasswordError] = useState(true);
```

Comment:
the values for these states are wrongly set for initialization. they should be set to false initially

Reply:
fixed initial value for errors state